src/s_conf.c

```diff
@@ -394,42 +395,46 @@ find_oper(char *name, char *username, char *sockhost, char *hostip)
     {
         if (aoper->legal == -1)
             continue;
+        if(!mycmp(name, aoper->nick))
+            nickmatch=1;
+        else
+            continue;
```

Comment:
It looks like `nickmatch` will always be 1 after this point. Is the `nickmatch` variable necessary?

Reply:
Good point, I would just use:
if(mycmp(name, aoper->nick))
    continue;

As there is no point to waste CPU for checking a CIDR for an entry that isn't the one we're looking for anyway...

Kobi.

Reply:
Damn. Good points. Thanks